app/src/models/progress.ts

```diff
@@ -104,6 +104,10 @@ export interface IRebaseProgress extends IProgress {
   readonly kind: 'rebase'
   /** The summary of the commit applied to the base branch */
   readonly commitSummary: string
+  /** The number of commits currently rebased onto the base branch */
+  readonly count: number
```

Comment:
How do you feel about updating the field names to indicate they having something to do with commits? Something similar to `rebasedCommitCount`

Reply:
Addressed in https://github.com/desktop/desktop/pull/7117/commits/322761dadb23373458ea6d0a729cde29326ebec6